### ISSUE TITLE
Increasing 8K Get Limit

### ISSUE THREAD

**Issue by ozgurhangisi** [swoole/swoole-src#1885]
Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a recipe for reproducing the error.

RFC for http 1.1 (https://tools.ietf.org/html/rfc7230#section-3.1.1) says that : 

Various ad hoc limitations on request-line length are found in practice.  It is RECOMMENDED that all HTTP senders and recipients support, at a **minimum, request-line lengths of 8000 octets**.

2. What did you expect to see?

RFC doesn't have any top limit but it gives the **minimum recommended size (8000 bytes)**. Swoole http server supports only requests up to 8K. I think this limit should be customizable or at least 64K. (Because all major browsers supports up to 64K URL length)

3. What did you see instead?

Swoole only support 8k Get request length and it can not be customizable.

4. What version of Swoole are you using (`php --ri swoole`)?

2.2.0

5. What is your machine environment used (including version of kernel & php & gcc) ? 

Aws t2 micro 1GM Ram, 1 CPU

6. If you are using ssl, what is your openssl version?

No

**Comment by twose** [comment]
I test it in chrome and my URL length is less than 8192 and it was unexpectedly failed...
why not post?

**Comment by ozgurhangisi** [comment]
Chrome supports more than 8KB url length. Now I tested with 12KB url on chrome (with nginx server) and it works fine. We create our URL with Java Script we don't manually add to the URL bar. 

We get many requests (about billions in a month) to our Load Balancers and we use Load Balancer logs (we have request URL in the log) to analyze the requests ( creating reports etc..). So It's not possible to see post data in load balancer log and we can not analyze our data if we use "post". (Also our Load Balancer supports up to 64KB Url length). (We are not using swoole as Load Balancer. Swoole servers are under our Load Balancer)

99% of our requests are under 8 KB but %1 of total request are bigger than 8KB but we can not get this data if we use swoole (Mostly important requests has many data.). 

**To keep url length in minimum (according to HTTP RFC) is not the best case for the real world apps.**

We tested swoole on our app with thousands of request in a second. Swoole's performance is really amazing (%50 less CPU usage and gets 2X more request for the real world app comparing with nginx/php 7.2.6 fpm) But 8K URL limit is blocking us to use swoole.
